owasp-security-logging-log4j/pom.xml

```diff
@@ -71,7 +79,7 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-core</artifactId>
-      <version>${log4j.version}</version>
+      <version>${log4j2.version}</version>
```

Comment:
Now you might wonder why this is here, this because its marked as a test-jar its not in the BOM thus we need the version.